spec/owner_content_resource_spec.rb

```diff
@@ -125,6 +125,7 @@
     original_serial.should_not == new_serial
   end
 
+  # TODO: This should be moved to the owner product resource spec tests
```

Comment:
Can it be moved as part of this PR? Otherwise whenever the owner_product card is picked up this test could be forgotten

Reply:
Done